### PR TITLE
templating hosts in ingress for docsportal

### PR DESCRIPTION
**Pull request by vladimirvshivkov**

### CODE REVIEW

File: playbooks/roles/document_hosting_k8s/defaults/main.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 document_hosting:
   swift_server: swift.example.com:443
   server_name: docs.example.com
+  beta_server_name: docs-beta.example.com
```
Comment (gtema) [inline playbooks/roles/document_hosting_k8s/defaults/main.yaml:4]: I guess we should better have something like "alt" - a general way for alternate or synonyms. Eventually we can convert server_name into domains (an array)

Comment (vladimirvshivkov) [reply playbooks/roles/document_hosting_k8s/defaults/main.yaml]: the array is less transparent than 2 specific servernames

Comment (gtema) [reply playbooks/roles/document_hosting_k8s/defaults/main.yaml]: issue is that from the webserver pov you can have as many hostnames as you wish

File: playbooks/roles/document_hosting_k8s/tasks/main.yaml [hunk 2]
```diff
@@ -290,10 +290,23 @@
...
+            http:
+              paths:
+                - path: "/"
+                  pathType: "Prefix"
+                  backend:
+                    service:
+                      name: "{{ document_hosting_svc_name }}"
+                      port:
+                        number: 8080
+          - host: "{{ document_hosting.server_name.1 }}"
```
Comment (gtema) [inline playbooks/roles/document_hosting_k8s/tasks/main.yaml:309]: this is a definitely wrong way to do this. Instead I suggest to convert ingress to ansible template and iterate using jinja (something like https://github.com/opentelekomcloud-infra/system-config/blob/main/playbooks/roles/grafana/templates/grafana-deployment.yaml.j2#L101) This way role should be capable of dealing with arbitrary count of hostnames.

**Comment by gtema** [comment]
This is now not necessary, as different approach has been deployed

**Pull request closed without merging**